seastar/src/lib.rs

```diff
@@ -16,3 +16,18 @@ pub(crate) use seastar_test_guard::acquire_guard_for_seastar_test;
 
 pub use config_and_start_seastar::*;
 pub use preempt::*;
+
+/// A macro intended for running asynchronous tests.
```

Comment:
Nit: put an empty comment line after this sentence. `cargo doc` uses the first block of text as a short summary of the item:

```rust
/// A macro intended for running asynchronous tests.
///
/// Tests are spawned in a separate thread.
/// This is done to ensure thread_local cleanup between them
/// (at the time of writing, Seastar doesn't do it itself).
```